src/index.js

```diff
@@ -1,15 +1,51 @@
-import _ from 'lodash';
 import './style.css';
-import Icon from './icon.png';
 
- function component() {
-   const element = document.createElement('div');
+const listContents = document.querySelector('.listContents');
 
-   // Lodash, now imported by this script
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+const data = [
+  {
+    descript: 'ghislain Developer',
+    completed: true,
+    index: 0,
+  },
+  {
+    descript: 'Mitahi hacker',
+    completed: true,
+    index: 1,
+  },
+  {
+    descript: 'zazu security',
+    completed: true,
+    index: 2,
+  },
+];
 
-   return element;
- }
+const list = document.createElement('ul');
+data.forEach((element) => {
+  const li = document.createElement('li');
+  const checkBox = document.createElement('input');
+  const p = document.createElement('p');
+  const i = document.createElement('i');
+  const label = document.createElement('label');
```

Comment:
[OPTIONAL]

- In my opinion, I think using innerHTML(i.e Writing DOM elements in javascript) is preferable instead of creating the elements individually in javascript as this will reduce the lines of code written and also make the code base cleaner 😄 .

e.g.

```
const innerHTML = 
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>;
```